models/models.js

```diff
@@ -8,13 +8,23 @@ exports.selectTopics = () =>{
 }
 
 exports.selectArticleId = (article_id) =>{
-   return db.query('SELECT * FROM articles WHERE article_id = $1', [article_id]).then((result)=>{
+   return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id]).then((result)=>{
       if (result.rows.length === 0){
          return Promise.reject({status:404, msg: 'article does not exist'})
       }
-      else if (!result){
-         return Promise.reject({status:400, msg: 'Bad request'})
-      }
       return result.rows[0]
    })
-}
\ No newline at end of file
+}
+
+exports.selectAllArticles = ()=>{
+   return db.query(`SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url,
+   COUNT (comments.article_id) AS comment_count
```

Comment:
This might make more sense to be an int rather than a string. Its a tiny refactor